ui/src/app/pages/register/register.component.ts

```diff
@@ -88,12 +89,17 @@ export class RegisterComponent extends NbRegisterComponent implements OnInit {
         company,
       }).subscribe(
         respReg => {
+          const first_name = this.user.fullname.split(' ')[0];
+          const last_name = this.user.fullName.replace(`${first_name} `, '');
+          _ps.getByKey(this._groupKey).send('updated', { custom_data: { first_name, last_name } });
+
           this.submitted = false;
 
           if (respReg.isSuccess()) {
```

Comment:
@iatdaitan outside of this issue's scope, register errors such as name or email already in use are not being correctly treated in this page, leading to a locked state where <Register> button no longer enables it self, even after changing inputs.

Reply:
@manrodrigues please, consider it during your test. I am going to open an issue to deal with those error enhancements.